### PR TITLE
System.contributors

### PR DESCRIPTION
**Pull request by godfreyd**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Category (leave one):

- Documentation

Added description about `system.contributors` table :

### CODE REVIEW

File: docs/en/operations/system_tables.md [hunk 1]
```diff
@@ -91,6 +91,46 @@ The `system.columns` table contains the following columns (the column type is sh
...
 
+## system.contributors {#system_contributors}
+
+Contains information about contributors. Arbitrary selection.
+
+Columns:
+
+- `name` (String) — Contributor name.
+
+**Пример**
```
Comment (hczhcz) [inline docs/en/operations/system_tables.md]: **Example**

File: docs/en/operations/system_tables.md [hunk 2]
```diff
@@ -91,6 +91,46 @@ The `system.columns` table contains the following columns (the column type is sh
 - `is_in_primary_key` (UInt8) — Flag that indicates whether the column is in the primary key expression.
 - `is_in_sampling_key` (UInt8) — Flag that indicates whether the column is in the sampling key expression.
 
+## system.contributors {#system_contributors}
+
+Contains information about contributors. Arbitrary selection.
```
Comment (alexey-milovidov) [inline docs/en/operations/system_tables.md]: Not an "Arbitrary selection".

Comment (alexey-milovidov) [reply docs/en/operations/system_tables.md]: All constributors in random order.

File: docs/en/operations/system_tables.md (hunk 1 above)
Comment (godfreyd) [reply docs/en/operations/system_tables.md]: OK!

File: docs/en/operations/system_tables.md (hunk 2 above)
Comment (godfreyd) [reply docs/en/operations/system_tables.md]: How?

Comment (godfreyd) [reply docs/en/operations/system_tables.md]: Done

**Comment by alexey-milovidov** [comment]
@godfreyd Look at the commits.

**Pull request merged**